sim/paladin/righteous_vengeance.go

```diff
@@ -0,0 +1,72 @@
+package paladin
+
+import (
+	"time"
+
+	"github.com/wowsims/sod/sim/core"
+	"github.com/wowsims/sod/sim/core/proto"
+)
+
+const RVTicks = 4
+
+func (paladin *Paladin) registerRV() {
+	if !paladin.hasRune(proto.PaladinRune_RuneCloakRighteousVengeance) {
+		return
+	}
+
+	paladin.RegisterAura(core.Aura{
+		Label:    "Righteous Vengeance",
+		Duration: core.NeverExpires,
+		OnReset: func(aura *core.Aura, sim *core.Simulation) {
+			aura.Activate(sim)
+		},
+		OnSpellHitDealt: func(aura *core.Aura, sim *core.Simulation, spell *core.Spell, result *core.SpellResult) {
+
+			if spell.Flags.Matches(SpellFlag_RV) && result.DidCrit() {
+				paladin.procRV(sim, result)
+			}
+		},
+	})
+
+	paladin.rv = paladin.RegisterSpell(core.SpellConfig{
+		ActionID:    core.ActionID{SpellID: 440675},
+		SpellSchool: core.SpellSchoolHoly,
+		//DefenseType: core.DefenseTypeMagic,
+		ProcMask:    core.ProcMaskEmpty,  //Not sure
+		Flags:       core.SpellFlagPureDot | core.SpellFlagIgnoreModifiers | core.SpellFlagIgnoreResists,
```

Comment:
Is the RV DoT guaranteed to be exactly 50% of the original value?

Reply:
It might be slightly more nuanced than that possibly, but for the most part seem yes it works like that to avoid double scaling with multiplier like Avenging Wrath possibly (caster side) or target side (nightfall / occult poison) 
![image](https://github.com/wowsims/sod/assets/5923944/dc24cd52-38c6-4bfb-b0d6-0a4958c9ec18)

Reply:
I went off of Ignore damage taken Mod. and ignore caster damage mod

Reply:
I thought so but in the comment below even though the ignite behavior matches, the scaling of the damage seems off. 5039 (actual) / 4191 (expected) = 1.202

Possible Explanations? well Occult poison is Applied and so is Sanctity aura.  (which adds up to 20%)
but so is Vengeance, which adds 15%

I'm not sure what flags would make it match.
Any Suggestions?

Lastly:
https://www.wowhead.com/classic/spell=440675/righteous-vengeance has the no partial resists flag and cannot miss flag but the logs indicate partial resists.